### ISSUE TITLE
Implement https://status.yorkie.dev

### ISSUE THREAD

**Issue by hackerwins** [yorkie-team/yorkie-team.github.io#97]
**Description**:

To easily check the operational status of the service, we need to create dedicated status pages for the service.

Before tackling these tasks, @lqez proposed the idea of adding a status page to show the service's current status.

Check out these references for inspiration:

https://status.io/
https://www.githubstatus.com/

This is moved from https://github.com/hackerwins/homepage-old/issues/11

**Why**:

**Comment by krapie** [comment]
I'm interested in this issue 😄

For status checking, all we need to do is just health check `api.yorkie.dev` right?

**Comment by hackerwins** [comment]
Yes, It would be good to check if Admin API and SDK API are alive for now.

**Comment by hackerwins** [comment]
https://stats.uptimerobot.com/otOOggryMR
